dist/allProjectsObject.js

```diff
@@ -1,10 +1,7 @@
 "use strict";
-//Create new Project object when new project is added.
-//Probable gonna need an array of projects that can be selected as well.
-//Project object needs title, and array of task objects that are stored within the project.
-//When tasks are added, create task Object, and store in current Project Object
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.Task = exports.Project = exports.AllProjects = void 0;
+const uuid_1 = require("uuid");
```

Comment:
dont use the require syntax if you can help it. use `import` at the top. also what is this actually doing?